pkger/parser_models.go

```diff
@@ -720,11 +724,14 @@ func (c *chart) properties() influxdb.ViewProperties {
 			Type:                       influxdb.ViewPropertyTypeMosaic,
 			Queries:                    c.Queries.influxDashQueries(),
 			ViewColors:                 c.Colors.strings(),
+			HoverDimension:             c.HoverDimension,
 			XColumn:                    c.XCol,
 			GenerateXAxisTicks:         c.GenerateXAxisTicks,
 			XTotalTicks:                c.XTotalTicks,
 			XTickStart:                 c.XTickStart,
 			XTickStep:                  c.XTickStep,
+			YLabelColumnSeparator:      c.YLabelColumnSeparator,
+			YLabelColumns:              c.YLabelColumns,
```

Comment:
Same questions about handling empty / unset for these new fields.

Reply:
No need. They will be interpreted correctly.